File: Ad-Hoc/__2000__/2355/2355.php
<?php

while($tempo = (int) readline()){
    if ($tempo == 0) break;

    $ale = (int) ceil(7.0 * $tempo / 90);
    $bra = (int) floor($tempo / 90.0);

    echo "Brasil $bra x Alemanha $ale\n";
}

File: Ad-Hoc/__2000__/2355/2355.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        while(in.hasNextLine()){
            int tempo = in.nextInt();
            if(tempo == 0) break;

            int ale = (int) Math.ceil(7.0 * tempo / 90);
            int bra = (int) Math.floor(tempo / 90.0);

            System.out.println("Brasil " + bra + " x Alemanha " + ale);

        }
        in.close();
    }
}